stages/org.osbuild.zipl.inst

```diff
@@ -25,6 +25,14 @@ SCHEMA_2 = r"""
     "kernel": {
       "type": "string"
     },
+    "kargs": {
+      "description": "Additional kernel argumets",
+      "type": "array",
+      "items": {
+        "description": "A single kernel command line option",
+        "type": "string"
+      }
+    },
```

Comment:
Assuming we can't find another way and we do need a mechanism for specifying addition kernel arguments let's make it explicit here that it's appending to the ones that are found in the system.

```suggestion
    "kernel_opts_append": {
      "description": "Additional kernel options to add to the discovered options",
      "type": "array",
      "items": {
        "description": "A single kernel command line option",
        "type": "string"
      }
    },
```

Also I think we'd want to break this out into a separate PR probably.